boards/FSM/Src/main.c

```diff
@@ -609,12 +610,12 @@ static void MX_TIM2_Init(void)
 
     /* USER CODE END TIM2_Init 1 */
     htim2.Instance               = TIM2;
-    htim2.Init.Prescaler         = 0;
-    htim2.Init.CounterMode       = TIM_COUNTERMODE_DOWN;
+    htim2.Init.Prescaler         = 1;
```

Comment:
are these prescalers here supposed to be 2? you mentioned how the encoder is incremented/decremented by 2

Reply:
Yea when testing 1 was what actually worked properly. Think it's because of this: 
![image](https://user-images.githubusercontent.com/28274296/158030345-ec13cb0c-6255-4b37-a663-28be3e5779ae.png)
So it's actually index 1 which is divide by 2. Is there a better way to show this in cube?

Reply:
Ah ok that's fine then

Reply:
Could you update this in the PR description?